ansible/playbooks/roles/opensearch/tasks/configure-ops.yml

```diff
@@ -3,19 +3,19 @@
 
 - name: Ensure snapshot folder exists
   file:
-    path: "{{ specification.paths.repo }}/"
+    path: "{{ specification.paths.ops_repo }}/"
```

Comment:
`ops` is a widely known acronym for operations and I don't see equivalent of `es` which was used for Elasticsearch since `os` is already taken for operating system, thus I would consider using full name `ops` -> `opensearch` for better readability.